tycho-baseline-plugin/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
```

Comment:
These jdt.core files were automatically added by importing the projects into the Eclipse SDK IDE. I can revert them if you want.

Reply:
I think thats fine.